pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -323,14 +324,23 @@ func (p *PoolManager) IsKubevirtEnabled() bool {
 	return p.isKubevirt
 }
 
-func (p *PoolManager) isRelatedToKubevirt(pod *corev1.Pod) bool {
+func (p *PoolManager) isRelatedToKubevirt(pod *corev1.Pod, networks []*multus.NetworkSelectionElement) bool {
 	if pod.ObjectMeta.OwnerReferences == nil {
+		log.V(1).Info("this pod has no OwnerReferences, so will be considered as a regular pod")
+		return false
+	}
+
+	// since the pod request may not have owned the vm namespace in this early state of vm making,
+	// we need to receive the namespace from the network annotation
+	namespace, ok := selectNamespaceFromNetworks(networks)
+	if !ok {
+		log.V(1).Info("this pod has more than 1 namespace in its secondary networks, so will be considered as a regular pod")
```

Comment:
How does this work? Any pod or VM can request networks from any number of namespaces.